README.md

```diff
@@ -16,7 +16,12 @@ the square brackets (for multiple-choice questions).
 Describe the purpose of a clearfix in CSS, and give an example of how to do it.
 
 Your Answer:
-```text
+```The purpose of clearfix is CSS is a way way for elements to clear its children elements.
+ex. .clearfix:after {
+  border: 100px;
+  background-color: red;
+}
+};
 ```
 
```

Comment:
Pretty much! Clearfix clears a container's floated elements, and also forces that container to actually contain those floated elements.

Does the example you included, however, actually do the job? Clearfix does make use of the `clear` property.

Useful blog post in case you need a refresher: https://css-tricks.com/all-about-floats/